assets/src/design-system/icons/action/index.js

```diff
@@ -37,5 +37,6 @@ export { ReactComponent as Preview } from './preview.svg';
 export { ReactComponent as Save } from './save.svg';
 export { ReactComponent as Search } from './search.svg';
 export { ReactComponent as Settings } from './settings.svg';
+export { ReactComponent as SwitchToDraft } from './switchToDraft.svg';
```

Comment:
You really ought to merge in the icon PR from #6311, as this icon is already imported, but in the proper size and bounding box. That'll also ensure that the icons scale properly relative to each other.

Reply:
Will do, I'm hoping it will get merged soon so I can keep the changed files of this PR under 100 😛 

Waiting for the design confirmation on the icons-vs-text for buttons, too.